lib/Service/TableService.php

```diff
@@ -75,12 +96,11 @@ public function findAll(?string $userId = null, bool $skipTableEnhancement = fal
 						$newSharedTables[] = $sharedTable;
 					}
 				}
-			} else {
-				$newSharedTables = [];
 			}
-		} catch (\OCP\DB\Exception $e) {
+		} catch (\OCP\DB\Exception|InternalError $e) {
 			$this->logger->error($e->getMessage());
```

Comment:
```suggestion
			$this->logger->error($e->getMessage(), ['exception' => $e]);
```

This way we also have the exception trace in the log